### ISSUE TITLE
Labs not auto-generating Discourse threads

### ISSUE THREAD

**Issue by neshoba78** [fablabbcn/fablabs#376]
Seems at least the last 20 lab applications submitted did not auto-generate threads in the Labs forum in Discourse. Can/Should we manually sync?

**Comment by openp2pdesign** [comment]
This was a bug that was later fixed, but in order to solve the issue we need to change directly the database, we need to carefully plan for this in 2018.

**Comment by bmcage** [comment]
Is my lab a victim of this? https://discuss.fablabs.io/c/labs does not have entry for IngegnoMakerSpace

I had mail 28/11 that our lab was approved, but still no link 
https://www.fablabs.io/labs/IngegnoMakerSpace up also.

**Comment by openp2pdesign** [comment]
@bmcage Nope, in your case there are two issues: a Referee Lab rejected you, and #391. My suggestion is to do the application again

**Comment by openp2pdesign** [comment]
This happened for around 23 `Labs`, mainly because

- There was already a `Lab` with the same name... the user should wait before creating multiple applications. Error: `{"action":"create_post","errors":["Title has already been used"]}`
- The description of the `Lab` should be added to Discourse without links in it. Error: `{"action":"create_post","errors":["Sorry, new users can only put 2 links in a post."]}`
- There is only one link and it is Facebook (not sure about this, but it seems a common problem). Error: `{"action":"create_post","errors":["Sorry you cannot post a link to that host."]}`

Strange Errors to be checked: 
-`{"errors":["You are not permitted to view the requested resource. The API username or key is invalid."],"error_type":"invalid_access"}`
- `{"errors":["You are not permitted to view the requested resource."],"error_type":"invalid_access"}`

We should check these problems in the code, and maybe write a script that "manually" create the topics when they are missing..
